projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -423,12 +423,16 @@ function get_element_styles_from_attributes( $attributes ) {
 	$submit_button_styles .= $style;
 	$email_field_styles   .= $style;
 
-	$button_spacing = get_attribute( $attributes, 'spacing', DEFAULT_SPACING_VALUE );
-	if ( true === get_attribute( $attributes, 'buttonOnNewLine' ) ) {
-		$submit_button_styles .= sprintf( 'margin-top: %dpx;', $button_spacing );
-	} else {
-		$submit_button_styles .= 'margin: 0px; '; // Reset Safari's 2px default margin for buttons affecting input and button union
-		$submit_button_styles .= sprintf( 'margin-left: %dpx;', $button_spacing );
+	$is_button_only_style = get_attribute( $attributes, 'className' ) === BUTTON_ONLY_CLASS_NAME;
```

Comment:
There should be PHP equivalent of the helper I mentioned above https://github.com/Automattic/jetpack/pull/33921/files/16f39dd56ac23f45bcbf557da1839d13faf1d8d5..35beec6f24a4831e79557ed3e04002b4fc8ac0d6#r1489104415

Reply:
I couldn't find it, asked the Crew folks p1707903492410909-slack-CBG1CP4EN.